axelor-sale/src/main/resources/i18n/messages_fr.csv

```diff
@@ -614,6 +614,7 @@
 "The client Partner is required and must be the same for all sale orders","Le client est requis et doit être le même pour toutes les cmdes clients",,
 "The client provided is not a customer of the given company: %s.","Le client fourni n'est pas un client de l'entreprise donnée: %s ",,
 "The company %s doesn't have any configured sequence for sale orders","La société %s n'a pas de séquence configurée pour les devis clients",,
+"The company configuration doesn't allow you to manage the exclusion or inclusion of taxes.","La configuration de l'entreprise ne permet pas de gérer l'exclusion ou l'inclusion des taxes.",,

```

Comment:
Change translation to :
La configuration de la société ne permet pas de gérer l'exclusion ou l'inclusion des taxes.